### PR TITLE
Tcio 42/footer

### PR DESCRIPTION
**Pull request by julika-gulchitai**

### CODE REVIEW

File: src/widgets/Footer/components/FollowUs.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const FollowUs = () => {
+  // const colorIcon = { 'colors.primary': '10' };
+  return (
+    <div className="flex flex-col mt-12">
+      <p className="font-azeret text-center font-azaret font-medium text-text-secondary">
+        FOLLOW US
+      </p>
+      <div className="flex flex-row justify-center items-center text-center gap-9 mt-12">
+        {/* <Icon icon="twitter" color={`${colorIcon}`} size={'w-[18px]'} /> */}
```
Comment (BohdanTaran) [inline src/widgets/Footer/components/FollowUs.tsx:15]: I see two similar rows (15,16) where you want to insert an icon, but this one is commented. Try to get rid of comments.

File: src/widgets/Footer/components/FollowUs.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import Icon from '../../../shared/components/Icon';
+
+// interface Props {
```
Comment (BohdanTaran) [inline src/widgets/Footer/components/FollowUs.tsx:3]: you do not use this props

File: src/widgets/Footer/components/OrderProcessing.tsx [hunk 3]
```diff
@@ -1,13 +1,13 @@
 const OrderProcessing = () => {
   return (
-    <div className="flex flex-col justify-start text-xs ">
-      <p className="font-roboto text-xs font-normal text-text-tertiary">
+    <div className="flex flex-col justify-star mb-13">
```
Comment (BohdanTaran) [inline src/widgets/Footer/components/OrderProcessing.tsx:3]: "justify-star" does not exist. Did you mean "justify-start" ?

**Review by BohdanTaran — CHANGES_REQUESTED** [review]
I think you should close this PR. You know that we have another git flow, for this reason it will be better if you make a copy of your current footer and make new branch from develop or just do git pull and after these manipulations you can paste your footer and send new PR. Now this PR contains 50+ commits with not latest updates.

File: src/widgets/Footer/components/FollowUs.tsx (hunk 1 above)
Comment (julika-gulchitai) [reply src/widgets/Footer/components/FollowUs.tsx:15]: Ok

**Comment by julika-gulchitai** [comment]
> I think you should close this PR. You know that we have another git flow, for this reason it will be better if you make a copy of your current footer and make new branch from develop or just do git pull and after these manipulations you can paste your footer and send new PR. Now this PR contains 50+ commits with not latest updates.

Yes, it will be better to close. I'll try nothing to be lost

File: src/widgets/Footer/components/FollowUs.tsx (hunk 2 above)
Comment (julika-gulchitai) [reply src/widgets/Footer/components/FollowUs.tsx:3]: Sure

**Pull request merged**